ci/loadWin.groovy

```diff
@@ -23,18 +23,30 @@ def check_tests(){
     status = bat(returnStatus: true, script: 'grep "       OK " win_test.log')
     if (status != 0) {
             error "Error: Windows run test failed ${status}. Expecting passed tests and no passed tests detected. Check win_test.log for details."
+    } else {
+        def passed = bat(returnStatus: false, returnStdout: true, script: 'grep "       OK " win_test.log | wc -l')
+        echo "Error: Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."
```

Comment:
echo "Success : Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."

Reply:
```suggestion
        echo "Success: Windows run test passed ${status}. ${passed} passed tests . Check win_test.log for details."
```